openml-generic-r/README.md

```diff
@@ -0,0 +1,24 @@
+OpenML Provider for generic R models
+------------------------------------------------------------
```

Comment:
Can we also help with "where" this implementation needs to be? (i.e., within a script folder, right?)

Reply:
I was not aware of this detail, but you are right. Details here: https://github.com/feedzai/feedzai-openml-r/blob/master/openml-r-common/src/main/java/com/feedzai/openml/r/GenericRModelLoader.java#L64

Will add to this PR

Reply:
this will be added by @krisztinaknagy on her following PR.